docs/modules/plugins/partials/generic-ui-steps.adoc

```diff
@@ -117,3 +117,89 @@ When I wait until element located `$locator` disappears
 Scenario: Wait for element disappearance
 When I wait until element located `name(welcome-image)` disappears
 ----
+
+=== Get text of context element
+
+==== *_Info_*
+
+Extracts text of a context element and saves it to a variable
```

Comment:
is it applicable for context **element** only? or is full page supported as well?

Reply:
added warning about fail if context element is not set